pandas/tests/dtypes/test_dtypes.py

```diff
@@ -1095,3 +1095,22 @@ def test_period_dtype_compare_to_string():
     dtype = PeriodDtype(freq="M")
     assert (dtype == "period[M]") is True
     assert (dtype != "period[M]") is False
+
+
+def test_compare_complex_dtypes():
+    # GH 28050
+    # https://github.com/pandas-dev/pandas/issues/28050
+    arr = np.arange(5).astype(np.complex128)
+    df = pd.DataFrame(arr)
```

Comment:
Could define this in one go. We don not need the https link

Reply:
Please put the arange call into the DataFrame() call, otherwise lgtm

Reply:
> Please put the arange call into the DataFrame() call, otherwise lgtm

Done - thanks for the lessons!